op-guide/configuration.md

```diff
@@ -11,6 +11,12 @@ TiDB, TiKV and PD are configurable using command-line flags and environment vari
 
 The default TiDB ports are 4000 for client requests and 10080 for status report.
 
+### `--advertise-address`
+
+- The IP address used to access the TiDB server
+- Default: ""
+- TiDB stores some information to PD, such as `advertise-address`. Make sure `advertise-address` is a public IP, such as 172.16.30.8. Otherwise, `advertise-address` is meaningless.
+
```

Comment:
`172.16.0.0/12` is a private IP network segment.

Reply:
I think you can reference kubernetes document about advertise-address https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/